R/iso_grid.R

```diff
@@ -1,11 +1,19 @@
 #' Create hexagonal grid
+#' 
+#' Create a hexagonal grid from an sf object
 #'
 #' @param x An sf object the convex hull of which will determine the hex grid
-#' @param ... Extra arguments passed to the grid generator (TBC)
+#' @param resolution The resolution of the hex grid
+#' @return An sf object
 #' @export
 #' @examples
 #' x = net_oldenburg_raw
 #' iso_grid(x)
-iso_grid = function(x, ...) {
-  sf::st_convex_hull(sf::st_union(x))
+iso_grid = function(x, resolution = 9) {
+  poly = sf::st_convex_hull(sf::st_union(x))
```

Comment:
Probably best for me to make the check in h3o, but there needs to be validation that the units are degrees

Reply:
Yeah... safe to assume EPSG:4326 I guess. We can mention that in the docs, if that would fix it?

Reply:
And at some point could shift to the whole `{rsgeo}` stack, but let's get something working first!